src/components/home/ThumbnailContent.tsx

```diff
@@ -61,33 +65,39 @@ const textCss = css`
   line-height: 150%;
 `;
 
-function LinkContent({ openGraph }: Pick<ContentThumbnailProps, 'openGraph'>) {
+function LinkContent({ openGraph, content }: Pick<ContentThumbnailProps, 'openGraph' | 'content'>) {
   // initial state를 og.url + og.image로 한 이유는
   // og.image가 안되는 경우가 상대주소로 작성이 되어 있어 영감탱 서버에 요청하기 때문
-  const [src, setSrc] = useState<string>(
-    openGraph && openGraph.url && openGraph.image ? openGraph.url + openGraph.image : ''
-  );
+
+  const [og, setOg] = useState<OpenGraph>();
+  const [src, setSrc] = useState<string>('');
+  const { checkIgonreOpenGraphHost, makeURLOpenGraph } = useIgonreOpenGrap();
+
+  useDidMount(() => {
+    setOg(checkIgonreOpenGraphHost(content) ? makeURLOpenGraph(content) : openGraph);
+    setSrc(og && og.url && og.image ? og.url + og.image : '');
+  });
 
   const onImageError = (e: SyntheticEvent<HTMLImageElement>) => {
-    if (!openGraph) return;
-    if (!openGraph.url) return;
-    if (!openGraph.image) return;
+    if (!og) return;
+    if (!og.url) return;
+    if (!og.image) return;
 
     // 두번째로 시도하는 og.image에서도 에러를 발생할 경우
     if (e.currentTarget.src === openGraph.image) return;
 
-    setSrc(openGraph.image);
+    setSrc(og.image);
   };
 
   return (
     <div css={linkWrapperCss}>
       <div css={linkImgWrapperCss}>
-        <img alt={`${openGraph?.url} thumbnail`} src={src} onError={onImageError} />
+        {src ? <img alt={`${og?.url} thumbnail`} src={src} onError={onImageError} /> : <></>}
```

Comment:
`src`가 없을 때는 alt 텍스트가 보이도록 하는 방향은 어떠실까요 !!
```suggestion
        <img alt={`${og?.url} thumbnail`} src={src} onError={onImageError} />
```
> 물론 이렇게 될 시 onImageError에 추가적인 대응이 필요해보이는데, 출시 이후에 전체적으로 핸들링해도 좋아요 ~!

```suggestion
        {src && <img alt={`${og?.url} thumbnail`} src={src} onError={onImageError} />}
```
혹은 이렇게도 나타낼 수 있을 거 같아요!! 개인적인 취향임다 ~!

Reply:
적용완료했습니다. 75832c8